pkg/inference/preset-inferences.go

```diff
@@ -56,13 +57,14 @@ var (
 	tolerations = []corev1.Toleration{
 		{
 			Effect:   corev1.TaintEffectNoSchedule,
-			Operator: corev1.TolerationOpEqual,
-			Key:      resources.GPUString,
+			Operator: corev1.TolerationOpExists,
```

Comment:
Why change from `corev1.TolerationOpEqual` to `corev1.TolerationOpExists`

Reply:
The origin one was different from the yaml file, and it will make the adapter cannot find the node